static/js/tools/map/bq_query_utils.test.ts

```diff
@@ -0,0 +1,485 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { StatMetadata } from "../../shared/stat_types";
+import { getNonPcQuery, getPcQuery } from "./bq_query_utils";
+import { PlaceInfo, StatVar } from "./context";
+
+const sv: StatVar = {
```

Comment:
since these are constants, you should uppercase the var names, e.g. TEST_SV

Reply:
done